### PR TITLE
ILS Fix-19/05/03

### PR DESCRIPTION
**Pull request by kztmtn**
UIAA 29 (Open issue #1168)
It's my first trial, so if there's anything, please advise.

### CODE REVIEW

File: Glideslope.json [hunk 1]
```diff
@@ -36274,15 +36274,15 @@
...
+    "bearing": 280.80, 
     "airportICAO": "UIAA", 
-    "elevation": 2272, 
+    "elevation": 2270, 
     "name": "GS", 
-    "longitude": 113.321222, 
+    "longitude": 113.32517319, 
     "associatedRunwayNumber": "29", 
     "frequency": 110300, 
     "glideslope": 3.5028054,
```
Comment (ItsW1ll) [inline Glideslope.json]: Please also change the glideslope to 2.5028080

**Comment by kztmtn** [comment]
So sorry,,, I did unnecessary revert.

**Comment by orangejuicie** [comment]
GP is 3.5.
So glideslope is 3.502808

http://www.caica.ru/common/AirInter/validaip/aip/ad/ad2/rus/uiaa/1-ad2-rus-uiaa-155.pdf

**Comment by kztmtn** [comment]
All fixed, thank you for your advice @ItsW1ll @orangejuicie

**Pull request merged**